spec/lib/pantry/helpers/request_verifier_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'rails_helper'
+
+RSpec.describe Pantry::Alexa::RequestVerifier do
+
+
+  describe ".verify" do
+
+    let(:body) do
+      " {\"session\":" +
+        "{\"sessionId\":\"SessionId.b4470d1c-67a0-4da2-98c7-853acead6773\",\"application\":{\"applicationId\":\"amzn1.ask.skill.057ceea6-1e3a-4edb-9a4f-f962f8cc0cac\"},\"attributes\":{},\"user\":{\"userId\":\"amzn1.ask.account.AEP3BKUW344CTHKJDR6P33Y2RJZL6AOSIIWPAEE23O6AIUHMLQ7VSG7NE2CHPDDDCMKF57M43ULCPZLG2U2CIN5NG7MWT6KCDVQS2PXDM5HQYN5J7A26JFPOT7E56W5NQ6S3GSDNINQQZEL5EHDFSXFBM3ST3M5CMK53IZHHQMBCYDA2EFD6OJL7HE6I7WGLSHSLMTN5I7YQDVQ\",\"accessToken\":null},\"new\":false}," +
+        "\"request\":" +
+        "{\"requestId\":\"EdwRequestId.9eaad9b4-48f7-4f6e-8ee9-42a4d7691d4e\",\"locale\":\"en-US\",\"type\":\"IntentRequest\",\"timestamp\":#{Time.now.to_i},\"intent\":{\"name\":\"Warning\",\"slots\":{}}},\"context\":{\"System\": {\"application\": {\"applicationId\": \"applicationId\"}, \"user\":{\"userId\": \"userId\",\"permissions\": {\"consentToken\": \"consentToken\"},\"accessToken\": \"accessToken\"},\"device\": { \"deviceId\": \"deviceId\",\"supportedInterfaces\":{} },\"apiEndpoint\":  \"apiEndpoint\"}}}"
```

Comment:
Might be worth taking a few minutes and making a ruby hash out of this (can dump this string into `JSON.parse` in a Rails console), so it's at least readable to other devs looking at the test setup.  Then wherever you're using this `body`, do `body.to_json`, which will turn it into a JSON string.